### PR TITLE
Remove Fixes from PR template

### PR DESCRIPTION
**Pull request by omnivion**
### Changes proposed in this PR:
- Updated the PR template

### Checklist 
- [X] All JUnit tests pass (`mvn clean verify`).
- [X] New unit tests written to cover new functionality.
- [X] Added and updated JavaDocs for non-test classes and methods.
- [X] No local design debt. Do you feel that something is "ugly" after your changes?
- [X] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
 ### Information
-- Fixes #_.
```
Comment (halprin) [inline .github/PULL_REQUEST_TEMPLATE.md]: Why are we removing this?

Comment (sfradkin) [reply .github/PULL_REQUEST_TEMPLATE.md]: I would think that "Fixes" is somewhat redundant to "JIRA story".  I usually just put the JIRA number here and the actual changes in the next section.

Comment (omnivion) [reply .github/PULL_REQUEST_TEMPLATE.md]: ^ what Scott said

Comment (saquino0827) [reply .github/PULL_REQUEST_TEMPLATE.md]: I could see this as a repeat of the Jira ticket's title or description. I usually just delete the Fixes and add the ticket number. I would be further inclined to just change it to say something like `Fixes or JIRA Ticket`. The reason why Fixes is there is if there are any developers who aren't on our JIRA working on fixes for this project. But I like it better in one line.

Comment (halprin) [reply .github/PULL_REQUEST_TEMPLATE.md]: This line was added originally for the users who create a PR that fixes a GitHub issue.  GitHub keys off `Fixes #23` which will close GitHub issue 23 as fixed once the PR is merged into `master`.

**Pull request closed without merging**